### DISCUSSION TITLE
Consider mapping /root to persistent storage

### DISCUSSION THREAD

**Discussion by garethhowell** [hassio-addons/addon-vscode#378] [Q&A] [answered]
Great addon!
One of the great aspects of VSC is its support for git. I have my config files in a git repository stored on my local server and on github. For VSC to seemlessly push/pull from a repository, there needs to be an ssh key setup. 

Currently, /root is only in the container, so if the container is rebuilt, /root is re-created and any ssh keys are lost.

Would it be possible to make /root persistent by mapping it?

**Comment by frenck** [comment, marked as answer]
> Currently, /root is only in the container, so if the container is rebuilt, /root is re-created and any ssh keys are lost.

That statement is incorrect. See:

![image](https://user-images.githubusercontent.com/195327/149797604-fc6a6fac-f118-4168-ab41-694ebd1353a9.png)
